Services/Dashboard/ItemsBlock/classes/class.ilPDSelectedItemsBlockGUI.php

```diff
@@ -400,7 +403,10 @@ public function confirmedRemove(): void
 
     public function confirmedUnsubscribe(): void
     {
-        $refIds = (array) ($this->http->request()->getParsedBody()['ref_id'] ?? []);
+        $refIds = $this->http->wrapper()->post()->retrieve(
+            'ref_id',
+            $this->refinery->to()->listOf($this->refinery->kindlyTo()->int())
+        );
         if (0 === count($refIds)) {
```

Comment:
```suggestion
        if ([] === $refIds) {
```

Reply:
I will tackle this in a refactoring branch if this is still the case in ILIAS 9, but i wanna keep the current refactroing separated from actual bugfixes.